docs/tutorials/kafka.ipynb

```diff
@@ -104,7 +106,9 @@
       "outputs": [],
       "source": [
         "!pip install tensorflow-io\n",
-        "!pip install kafka-python"
+        "!pip install kafka-python\n",
+        "!pip install sklearn\n",
+        "!pip install pandas"
```

Comment:
@dalelane `sklearn` and `pandas` are pre-installed in colab. You can remove these lines.